### PR TITLE
Frontpage V2

### PR DESCRIPTION
**Pull request by ziccardi**
## Motivation
Created a frontpage for the new website

An alternative proposal to #1076, more appealing and modern

### CODE REVIEW

**Comment by Aindriunannerb** [comment]
Looks very nice, a higher resolution logo to replace the one in the bottom left of the page would be nice. It would be nice to also keep that info diagram from the first version of the frontpage, But I'm not sure how/where you would implement it.

File: website/website/pages/en/index.js [hunk 1]
```diff
@@ -92,117 +28,60 @@ class Index extends React.Component {
...
-
-    const TryOut = () => (
-      <Block id="try">
+    const Features = () => (
+        <div>
+      <Block layout="threeColumn">
+          {[
+              {
+                  content: 'A server written in Java which allow to send push notifications to different mobile operating systems and web application',
+                  image: `${baseUrl}img/notification.svg`,
```
Comment (Aindriunannerb) [inline website/website/pages/en/index.js]: possible to make the image also a link? This is what I clicked first. If its a pain, would making them into cards be a better option?

Comment (ziccardi) [reply website/website/pages/en/index.js]: Done!

**Comment by ziccardi** [comment]
@Aindriunannerb I removed the icon from the footer since it didn't add any value to the website. The diagram is still in the repo, we will probably add it back in another documentation page, maybe an intro one.
I should have fixed all the issues reported: mind giving another look?

File: website/website/pages/en/index.js [hunk 2]
```diff
@@ -47,30 +47,35 @@ class Index extends React.Component {
             content: 'A Java library that allows the integration of the UnifiedPush Server into your native Android application',
             image: `${baseUrl}img/java.svg`,
             imageAlign: 'top',
+            imageLink: '/docs/configuring_clients#android',
```
Comment (Aindriunannerb) [inline website/website/pages/en/index.js]: Awesome stuff, much more smoother user experience!

File: website/docs/introduction/how-it-works.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+id: how-it-works
+title: How the UnifiedPush Server works
+---
+
+The _AeroGear UnifiedPush Server_ can be seen as a broker that distributes push messages to different 3rd party Push Networks. The graphic below gives a little overview:
+
+![Load Firebase Assistant](/img/ups-overview.png)
```
Comment (Aindriunannerb) [inline website/docs/introduction/how-it-works.md]: is it possible to make the image a little big bigger?, the text is small and the light blue colour on the white background makes it a little bit hard to read.

Comment (ziccardi) [reply website/docs/introduction/how-it-works.md]: Don't think so... It already occupies the whole available width

**Pull request merged**